plugins/README.md

```diff
@@ -583,6 +589,34 @@ To update host to empty parameters, you can use this example.
     tls_connect: unencrypted
     proxy: ''
     inventory_mode: disabled
+    interfaces: []                        # Read important note in this example
+  vars:
+    ansible_network_os: zabbix.zabbix.zabbix
+    ansible_connection: httpapi
+    ansible_user: Admin
+    ansible_httpapi_pass: zabbix
+```
+
+**IMPORTANT**: If you want to clear templates and interface on the host, but the template contains items that use this interface, then you need to perform this operation in 2 tasks: first clear templates, then clear interfaces. If the template contains items that do not use an interface, clearing the template and removing interfaces can be done in one task.
```

Comment:
If you want to clear templates and interfaces on the host, but the template contains items that use this interface, then you need to perform this operation in two steps: first, clear the templates, then clear the interfaces. If the template contains items that do not use an interface, clearing the template and removing the interfaces can be done in one step.